examples/hello-world/step-by-step/cifar10/cse/cse.ipynb

```diff
@@ -27,7 +27,7 @@
     "We will be using the [Client API FL code](../code/fl/train.py) trainer converted from the original [Training a Classifer](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html) example.\n",
     "\n",
     "Key changes when writing a FL code to support multiple tasks:\n",
-    "- When using the default `launch_once` Client API Launcher Executor parameter, we encapsulate our code in `while flare.is_running():` loop so we can call `flare.receive()` and perform various tasks. This is useful when launching everytime would be inefficient, such as when having to perform data setup every time.\n",
+    "- When using the default `launch_once` parameter of "SubprocessLauncher", we encapsulate our code in `while flare.is_running():` loop so we can call `flare.receive()` and perform various tasks. This is useful when launching everytime would be inefficient, such as when having to perform data setup every time.\n",
```

Comment:
The "SubprocessLauncher" quotes might be causing an "invalid json" rendering issue

Reply:
OH, that's why these colors, I was wondering why,  let me remove those quotes
Thanks